docs/topics/api/accounts.rst

```diff
@@ -2,64 +2,83 @@
+    :>json string name: The name chosen by the user, or the username if not set.
+    :>json array addons_listed: Array holding the :ref:`addons <addon-detail-object>` the user has developed and are listed on this website.
+    :>json float averagerating: The average rating for the `addons_listed`.
+    :>json string|null biography: More details about the user.
+    :>json string|null homepage: The user's website.
+    :>json string|null location: The location of the user.
+    :>json string|null occupation: The occupation of the user.
+    :>json string picture_url: URL to a photo of the user, or `/static/img/anon_user.png` if not set.
+    :>json string|null picture_type: the image type (only 'image/png' is supported) if a user defined photo has been provided, or none if no photo has been provided.
+    :>json array reviews: Array holding the :ref:`reviews <review-detail-object>` the user has written.
+    :>json boolean is_addon_developer: The user has developed and listed add-ons on this website.
+    :>json boolean is_artist: The user has developed and listed themes on this website.
 
-        {
-            "username": "bob",
-            "display_name": "bob",
-            "email": "a@m.o",
-            "biography": "Some biography",
-            "deleted": false,
-            "display_collections": false,
-            "display_collections_fav": false,
-            "homepage": "https://a.m.o",
-            "location": "Vancouver",
-            "notes": null,
-            "occupation": "",
-            "picture_type": "",
-            "picture_url": "/static/img/anon_user.png",
-            "read_dev_agreement": "2015-11-20T18:36:12",
-            "is_verified": true
-        }
 
     :statuscode 200: profile found.
     :statuscode 400: an error occurred, check the `error` value in the JSON.
-    :statuscode 401: authentication failed.
+    :statuscode 404: no profile with that user id.
 
 
 .. important::
 
-    * Biography can contain HTML, or other unsanitized content and it's the
-      responsibiliy of the client to clean it appropriately before display.
+    * `Biography` can contain HTML, or other unsanitized content, and it is the
+      responsibiliy of the client to clean and escape it appropriately before display.
+
+
+------------
+Self Profile
+------------
+
+.. _`self-profile`:
+
+If you authenticate and access your own profile (either by specifing your own user_id, or omiting it) the following additional fields are returned.
+If you have `Users:Edit` permission you will see these extra fields for all user profiles.
+
+.. http:get:: /api/v3/accounts/profile/
+
+    .. _self-profile-object:
+
+    :>json string email: Email address used by the user to login and create this account.
+    :>json string|null display_name: The name chosen by the user.
+    :>json array collections: Array holding the collections the user has created.
```

Comment:
Again, a separate request is preferable here IMHO since there might be a ton of collections returned. Just build the minimal API required to do this. (with filtering like the reviews API, but only implement filtering on `user`, and raise an error if the filter parameter is not present)